test/integration/targets/aws_acm/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/aws
+shippable/aws/group2
+aws_acm_info
+aws_acm_facts
```

Comment:
no need to test `aws_acm_facts` (if there is it should be `_aws_acm_facts`)

Reply:
Fixed